src/docs/UBUNTU_BUILD.md

```diff
@@ -0,0 +1,42 @@
+# Local development build [Ubuntu Linux / WSL2]
+
+__Ensure you have the dependencies.__
+
+## _Install Python_ 
+```
+sudo apt update
+sudo apt install python3 python3-pip python3-virtualenv
+```
+
+## _Setup Python virtual environment and Install PlatformIO_
+```
+virtualenv -p python3 .venv
+source .venv/bin/activate
+pip install platformio
+```
+
+## _Checkout Marlin_
+```
+git clone git@github.com:MarlinFirmware/Marlin.git Marlin -b bugfix-2.0.x --depth 1
+cd Marlin
+git checkout bugfix-2.0.x
```

Comment:
The -b already set the branch to bugfix-2.0.x.

Reply:
oops... fixing

Reply:
Fixed.